.travis.yml

```diff
@@ -10,6 +10,8 @@ cache:
 before_script:
   - npm install -g coffeelint
   - npm install -g codecov
+  - ./bin/validate_cql4browsers.sh
```

Comment:
This is great.  You know what would also be great?  A similar script to check if the test data ELM representations are up to date (by re-running the `build-test-data` and checking for diffs).

Not in this PR though.

Reply:
That would definitely be useful but will be a bit more work with needing a JDK set up in Travis.

Reply:
Yeah, like I said.  I'm not expecting it in this PR.  I'm not even expecting you (or anyone on Bonnie) to do it at all.  Just throwing it out there because I thought of it -- and it's something I always check for manually when doing a code review.